frontend/timesheet/timesheetController.js

```diff
@@ -1155,6 +1200,52 @@ angular.module('mifortTimesheet.timesheet', ['ngRoute', 'constants'])
                 });
             };
 
+            $scope.saveAllHours = function () {
+                //Notification({message: 'Button clicked!', delay: 5000}, 'warning');
+
+                const dailyHours = $scope.filteredLogs;
+
+                let status = 0;
```

Comment:
Numbers for statuses gives bad code readability. Refactor status to be a string variable.